ark/mibi/tiling_utils.py

```diff
@@ -290,7 +309,7 @@ def set_tiling_params(fov_list_path, moly_path, tma=False):
 
     # read in the fov list data
     with open(fov_list_path, 'r') as flf:
-        fov_tile_info = json.load(flf)
+        fov_list_info = json.load(flf)
```

Comment:
I'm not sure if we need `set_tiling_params` to support TMAs anymore

Reply:
Changed my mind. Cleaning up the tiling-specific functions can be for the next PR. For this one, just make sure that all of the TMA functions are updated for the new, simplified logic

Reply:
Since TMAs only contain 2 FOVs, one defining the top-left FOV and one defining the top-right FOV, I think it makes sense. However, if we nuke TMA support in `set_tiling_params`, we will still need a way to generate the FOV list (the automatically-generated FOVs) without the `tiling_params` returned by this function for TMAs. See the next comment for an idea.